packages/merkle-distributor/hardhat.config.ts

```diff
@@ -0,0 +1,20 @@
+import "@nomiclabs/hardhat-web3";
+import "@nomiclabs/hardhat-truffle5";
+
+import { getHardhatConfig } from "@uma/common";
+import path from "path";
+
+const coreWkdir = path.dirname(require.resolve("@uma/core/package.json"));
+const packageWkdir = path.dirname(require.resolve("@uma/merkle-distributor/package.json"));
+
+const configOverride = {
+  paths: {
+    root: coreWkdir,
+    sources: `${coreWkdir}/contracts`,
+    artifacts: `${coreWkdir}/artifacts`,
+    cache: `${coreWkdir}/cache`,
+    tests: `${packageWkdir}/test`
```

Comment:
nit: do you want to run the validation-test directory as part of the hardhat tests?

Reply:
we _could_. issue is these tests require Cloudflare keys + an Infura API Key as they ensure that the end to end data retrieval works as expected. would you like this kind of test to be in CI?

Reply:
I wonder if you can easily separate out the unit tests that don't require secrets. Would be good to have as much test coverage in CI as possible

Reply:
@nicholaspai agreed. It would be nice to factor our ci that way.

@chrismaree didn't realize these required API keys. Not running it in ci makes sense, at least given our current setup!